helm/ovn-kubernetes/values-no-ic.yaml

```
@@ -181,3 +181,8 @@ monitoring:
   # in defining that.
   commonServiceMonitorSelectorLabels:
     release: kube-prometheus-stack
+  # -- deploy ServiceMonitors for service discovery using the Prometheus Operator
+  enableServiceMonitor: false
+  # -- deploy PrometheusRules for specific metric collection using the Prometheus Operator
+  enablePrometheusRule: false
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)